symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/OboUserService.java

```diff
@@ -17,55 +17,70 @@
 public interface OboUserService {
 
   /**
-   * {@link UserService#searchUserByIds(List, Boolean)}
+   * {@link UserService#searchUserByIds(List, Boolean, Boolean)}
    *
    * @param uidList     List of user ids
    * @param local       If true then a local DB search will be performed and only local pod users will be
    *                    returned. If absent or false then a directory search will be performed and users
    *                    from other pods who are visible to the calling user will also be returned.
+   * @param active      If not set all user status will be returned,
+   *                    if true all active users will be returned,
+   *                    if false all inactive users will be returned
    * @return Users found by user ids
    * @see <a href="https://developers.symphony.com/restapi/reference#users-lookup-v3">Users Lookup V3</a>
    */
-  List<UserV2> searchUserByIds(@Nonnull List<Long> uidList, @Nonnull Boolean local);
```

Comment:
we lost the @nonull annotation here, is it normal?

Reply:
I think @nonnull was added by mistake when the service was developed, that's why I removed it.
https://github.com/symphonyoss/symphony-api-spec/blob/master/pod/pod-api-public.yaml#L1456